app.js

```diff
@@ -1,25 +1,51 @@
 // IMPORT PACKAGES
 // Here you should import the required packages for your Express app: `express` and `morgan`
```

Comment:
Also we can import articles data as a variable 
```suggestion
// Here you should import the required packages for your Express app: `express` and `morgan`
const projects = require('./data/projects.json')
const articles = require('./data/articles.json')
```